### ISSUE TITLE
enter key confilct with chinese typewriting

### ISSUE THREAD

**Issue by rainssong** [felixbade/chatgpt-web-ui#49]
keyCode == 13
will fix this

```
    textbox.addEventListener('keydown', (event) => {
        if (event.keyCode == 13 && !event.ctrlKey && !event.altKey && !event.shiftKey) {
            event.preventDefault()
            submitMessageForm()
            updateTextareaSize(textbox)
        }
    })
```

**Comment by felixbade** [comment]
I would like to understand this issue so I can replicate it on my computer. Is it related to pinyin?

[`KeyboardEvent.keyCode`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) is deprecated, but this might be fixable with [`KeyboardEvent.key`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key).

**Comment by rainssong** [comment]
yep, when use pinyin, the enter means confirm english character（instead chinese character）

and event.key=='Enter' works

**Comment by felixbade** [comment]
`event.key === 'Enter'` seems to fix this issue on Firefox but not on Chrome. Might need to use the deprecated keyCode after all.

**Comment by felixbade** [comment]
Fixed in 375b3c0 and cd48078
